gm4_metallurgy/data/gm4_vibro_shamir/functions/shock/user.mcfunction

```diff
@@ -0,0 +1,22 @@
+# @s = player who used vibro
+# run from shock/activate
+
+# set up player
+tag @s add gm4_vibro_user
+scoreboard players operation $damage gm4_vibro_shock = @s gm4_vibro_shock
+
+# calculate damage dealt to mobs
+execute as @e[team=!invalid_team,type=!armor_stand,tag=!gm4_vibro_user,distance=..6] at @s run function gm4_vibro_shamir:shock/calculate
```

Comment:
That's used to check for a living mob, since the team selector only works for mobs and not other entities. This is used in liquid tanks, tho maybe we should prefix it `gm4_invalid_team` incase some outside module has a team named that?

Reply:
I've updated it to use a prefixed team name